### PR TITLE
remove aligned width from wrapper.

### PR DESCRIPTION
**Pull request by ziquanyu**
removed aligned width and tested on nexus6.

### CODE REVIEW

**Comment by ziquanyu** [comment]
@emilwestergren  Can you review this when you have time. Thanks!

File: nexus5/VidhanceProcessor.cpp [hunk 1]
```diff
@@ -6,16 +6,14 @@
...
 #define LOG_TAG "VidhanceProcessor"
-
 namespace imint {
+
 CameraWrapper* CameraWrapper::createMethod(const char* cameraId){
 	return new VidhanceProcessor(cameraId);
 }
 
 stride_t VidhanceProcessor::getStride(const sp<GraphicBuffer>& buffer) {
-	stride_t stride;
```
Comment (emilwestergren) [inline nexus5/VidhanceProcessor.cpp]: This change should not be part of this PR. Create a PR first where you adapt Nexus 5 code and we can merge that first. Then you can have a new PR where you only remove aligned width code.

Comment (ziquanyu) [inline nexus5/VidhanceProcessor.cpp]: okay, will remove it.
